Felienne/hedy
# Fix log search results not displayed properly

**Move the admin and class program log search to a separate page**

**Description**

_Split the log search form from the statistics page both for admins and teachers_
_Ensure the search results are displayed correctly and the load more data button appears correctly_

**Fixes #2983**

**How to test**

* Log in as an admin and navigate to the /admin page. Ensure that the 'Statistics' and 'Logs' button navigate correctly to the statistics and log search pages, respectively. Log in as a teacher and navigate to /for-teachers/class/{class-id} page and check that the 'statistics' and 'programs' buttons navigate to the class statistics and class log search pages, respectively.
* Unfortunately, the search log can only be tested on alpha. Once deployed to alpha, go to the log search form and make a query. Make the query span over a large period, so that the results allow for multiple pages. Then ensure
    - the button 'Load more' appears correctly on the screen. 
    - the program code is wrapped, so there is nothing overflowing.
    - the green copy code button in the right corner of every program code cell is visible and successfully copies the program code to the clipboard.
